samples/boards/nrf52/mesh/onoff_level_lighting_vnd_app/src/common.h

```diff
@@ -8,6 +8,8 @@
 #ifndef _COMMON_H
 #define _COMMON_H
 
+/* #define ENABLE_SMP_SVR */
```

Comment:
This should be an application level Kconfig option, no? And in fact, why not just key it off of whether or not MCUMGR is enabled?